test/res.sendFile.js

```diff
@@ -286,6 +286,82 @@ describe('res', function(){
       .get('/')
       .expect(200, 'got it', done);
     })
+    
+    it('should set etag according to app settings', function (done) {
+       var app = express();
+        
+       app.disable('etag');
+       app.use(function (req, res) {
+            res.sendFile(path.resolve(fixtures, 'name.txt'));
+        });
+     
+       request(app)
+        .get('/')
+        .end(function(err, res) {
+            if (err) {
+                return done(err);
+            }
+            assert.equal('etag' in res.headers, false);
+            done();
+        });
+    })
+    
+    it('should set etag according to app settings, with input args priority', function (done) {
```

Comment:
This test should be in the group with the above test, not in the base res group.